api/controller/contestants.go

```diff
@@ -93,11 +95,14 @@ func fetchContestansFromCSV(resp *render.Render, w http.ResponseWriter) ([]model
 			Age:          age,
 			CurrentCity:  line[4],
 			CurrentScore: score,
+			Bio:          utils.FetchInfo(id),
```

Comment:
Could there be a way to only call this api once instead of one per contestant? If you have a large data set, then you would end up having a loooot of calls to the api, which then would result in significant latency when calling your endpoints